cmd/gpu-feature-discovery/main.go

```diff
@@ -99,6 +100,12 @@ func main() {
 			Usage:   "Use NFD NodeFeature API to publish labels",
 			EnvVars: []string{"GFD_USE_NODE_FEATURE_API", "USE_NODE_FEATURE_API"},
 		},
+		&cli.StringFlag{
+			Name:    "mode",
+			Value:   "auto",
+			Usage:   "Select GFD mode between 'auto','nvml','tegra' or 'vfio'",
+			EnvVars: []string{"GFD_MODE"},
```

Comment:
```suggestion
			EnvVars: []string{"MODE", "GFD_MODE"},
```
or even just `MODE`?

Reply:
Fixed